app/models/setting.rb

```diff
@@ -1,6 +1,6 @@
 # Model class for site settings
 class Setting < ApplicationRecord
   def self.get( name )
-    find_by( name: name ).value
+    find_by( name: name )&.value
```

Comment:
Using `where(...).pluck(:value)` ought to avoid needing the safe navigation operator.